url-parameters/tests.json

```diff
@@ -0,0 +1,39 @@
+{
+    "trackingParameters": {
+        "name": "Tracking parameters are able to be removed from a URL",
+        "desc": "Those tests use config_reference.json",
+        "referenceConfig": "config_reference.json",
+        "tests": [
+            {
+                "name": "Remove utm tracking parameters from URL",
+                "testURL": "http://www.example.com/test.html?utm_source=test&utm_medium=test&utm_campaign=test&utm_term=test&utm_content=test",
+                "expectURL": "http://www.example.com/test.html",
+                "exceptPlatforms": []
+            },
+            {
+                "name": "Remove fbclid parameter and retain other parameters",
+                "testURL": "http://www.example.com/test.html?fbclid=test&q=test&id=5",
+                "expectURL": "http://www.example.com/test.html?q=test&id=5",
+                "exceptPlatforms": []
+            },
+            {
+                "name": "Retain all non tracking parameters",
+                "testURL": "http://www.example.com/test.html?q=test&id=5",
+                "expectURL": "http://www.example.com/test.html?q=test&id=5",
+                "exceptPlatforms": []
+            },
+            {
+                "name": "Don't remove parameters because domain is in exceptions list",
+                "testURL": "http://example2.com/test.html?gclid=test&fbclid=test",
+                "expectURL": "",
+                "exceptPlatforms": []
+            },
+            {
+                "name": "Remove 2 tracking parameters and retain other parameters",
+                "testURL": "http://www.example.com/test.html?utm_source=test&utm_medium=test&q=test&id=5",
+                "expectURL": "http://www.example.com/test.html?q=test&id=5",
+                "exceptPlatforms": []
+            }
```

Comment:
some more edge-cases:

1. param is in the domain name/path like `http://gclid.test.com/fbclid=123`
2. param has no value like `https://some.test.com?fbclid&gclid` (not actually sure what should happen in this case?)
3. params is ALMOST a match like `https://some.test.com?fbclid1=123`